components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -185,7 +185,11 @@ function Main() {
   })
 
   return (
-    <main className={styles.main} ref={setMainElement}>
+    <main className={classnames({
+      [styles.main]: true,
+      [styles.mainPanel]: !aiChatContext.isStandalone,
+      [styles.mainMobile]: aiChatContext.isMobile
```

Comment:
We actually have an `isMobile` class on the full page component already - I wonder if we could let the variables flow from there?